packages/manager/modules/billing-components/src/components/commitment/translations/Messages_fr_FR.json

```
@@ -27,5 +27,7 @@
   "billing_commitment_success_purchase": "Veuillez finaliser le paiement de votre <a href=\"{{url}}\" target=\"_blank\">bon de commande</a>.",
   "billing_commitment_error": "Une erreur est survenue.",
   "billing_commitment_summit": "Valider",
-  "billing_commitment_summary_description_end": "Après le {{ endDate }} votre service sera tacitement réengagé pour une durée similaire à votre dernier engagement."
+  "billing_commitment_summary_description_end": "Après le {{ endDate }} votre service sera tacitement réengagé pour une durée similaire à votre dernier engagement.",
+  "billing_commitment_savings": "Économisez un total de {{ price }}",
```

may be better, to check:
```suggestion
  "billing_commitment_savings": "Économiser un total de {{ price }}",
```

---

@radireddy any update regarding @mohammed-zahaf's suggestion?
Do we have to move forward with actual diff or changes will be introduced?

---

Hello @antleblanc, i discuss on it with the reporter he will check with cntent.

---

So, i received reponse it must be :
  "billing_commitment_savings": "Économiser un total de {{ price }}",